_pages/schedule.json.liquid

```diff
@@ -0,0 +1,88 @@
+---
+permalink: /schedule.json
+layout: null
+---
+{
+  "schedule": {
+    "version": "0.1",
+    "base_url": "https://laborluxeria.github.io/winterchaos2022/schedule/",
+    "conference": {
+      "acronym": "winterchaos-2022",
+      "title": "Winterchaos",
+      "start": "2022-12-28",
+      "end": "2022-12-30",
+      "daysCount": 3,
+      "timeslot_duration": "00:05",
+      "time_zone_name": "Europe/Zurich"
+    },
+    "rooms": [
+      {
+        "name": "Winterchaos",
+        "guid": "99411f15-7335-4d36-abac-3ff0419ec954",
+        "description": "Talks at LuXerias and LABOR Luzerns Winterchaos 2022",
+        "capacity": null
+      }
+    ],
+    "days": [
+      {% assign days = site.sessions | group_by: "day" | sort: "name" %}
+      {% assign all_sessions = days | map: "items" %}
+      {% assign all_speakers = all_sessions | map: "speaker" | join: ", " | split: ", " | uniq %}
+      {% assign i = 1 %}
+      {% for day in days %}
+        {
+          "index": {{ i }},
+          "date": "{{ day.name }}",
+          "day_start": "{{ day.name }}T00:00:00+01:00",
+          "day_end": "{{ day.name }}T00:00:00+01:00",
+          "rooms": {
+            "Winterchaos": [
+              {% assign sessions = day.items | where: "public", true | sort: "start" %}
+              {% for session in sessions %}
+              {
+                "id": "{{ session.talk_id }}",
+                "guid": "8e3616f7-df59-4f67-847d-783fea4a{{ session.talk_id }}",
```

Comment:
Regarding the open question: Using GUID4 here is fine, I think. Let's do it this way.

Not sure if it was obvious, but the the talk id is DDHH (DD = day of the month, HH = hour of the day). So if a talk starts Dec 28 at 16:00, it's talk_id is 2816. I rely on this for sorting the talks in the collection.

This means that if we move a talk, this guid will change. Which I think is fine? If that's not okay, then we should just store a unique random generated GUID in each talk file.

Reply:
Talk GUIDs *should* stay the same for each talk, so we (as in, the VOC pipeline) can accurately detect wether talks are new or just changed. If possible, we'd prefer to have static GUIDs.

Reply:
Thanks for the heads up! We will introduce static IDs then